src/components/sidebarLeft/index.ts

```diff
@@ -84,6 +85,18 @@ export class AppSidebarLeft extends SidebarSlider {
       });
     };
 
+    const onNewGeoGroupClick = () => {
```

Comment:
this part can be removed - it was only needed during testing. I'll remove it in the next commit.